phys/module_diag_misc.F

```diff
@@ -28,13 +28,24 @@ SUBROUTINE diagnostic_output_calc(                                 &
                      ,I_ACSWUPB,I_ACSWUPBC,I_ACSWDNB,I_ACSWDNBC       & ! Optional
                      ,I_ACLWUPT,I_ACLWUPTC,I_ACLWDNT,I_ACLWDNTC       & ! Optional
                      ,I_ACLWUPB,I_ACLWUPBC,I_ACLWDNB,I_ACLWDNBC       & ! Optional
+                     ,athmp, aqvmp                                    &
+                     ,athcu, aqvcu, aucu, avcu                        &
+                     ,athsh, aqvsh, aush, avsh                        &
+                     ,athbl, aqvbl, aubl, avbl                        &
+                     ,athralw, athrasw                                &
+                     ,h_diabatic, qv_diabatic                         &
+                     ,rthcuten, rqvcuten, rucuten, rvcuten            &
+                     ,rthshten, rqvshten, rushten, rvshten            &
+                     ,rthblten, rqvblten, rublten, rvblten            &
+                     ,rthratenlw, rthratensw                          &
                      ,dt,xtime                                        &
                      ,diag_print                                      &
                      ,bucket_mm, bucket_J                             &
                      ,prec_acc_c, prec_acc_nc, snow_acc_nc            &
                      ,snowncv, prec_acc_dt, curr_secs2                &
                      ,history_interval                                &
                      ,itimestep                                       &
+                     ,cu_used, shcu_used, acc_phy_tend                &
```

Comment:
cu_used and shcu_used are already available?

Reply:
@dudhia Yes. They are derived namelists, essentially.